Wrappers/Python/test/test_DataProcessor.py

```diff
@@ -1451,6 +1451,19 @@ def test_CofR_xcorrelation(self):
         ad_out = corr.get_output()
         self.assertAlmostEqual(6.33, ad_out.geometry.config.system.rotation_axis.position[0],places=2)              
 
+        # test there is no error with limited angle data, and target angle below 180
```

Comment:
I can see we have the following tests:

0-358.8 degrees data with starting index 0 and tol 0.1
0-358.8 degrees data with starting index 67 and tol 0.1
1-178.8 degrees data with starting index 0 and tol 3
1-178.8 degrees data with starting index 0 and tol 1 (testing failure)

I think it would be good to have similar tests with:
 Data which is -360 to 0 degrees
 Data which is -1 to -178.8

Reply:
I've simplified the tests a bit and have included these ranges